reference/5.1/Microsoft.PowerShell.Utility/Invoke-Expression.md

```diff
@@ -42,58 +55,64 @@ Handles  NPM(K)    PM(K)      WS(K) VM(M)   CPU(s)     Id   ProcessName
 ...
 ```
 
-This example demonstrates the use of **Invoke-Expression** to evaluate an expression.
-Without **Invoke-Expression**, the expression is printed, but not evaluated.
+This example demonstrates the use of `Invoke-Expression` to evaluate an expression.
+Without `Invoke-Expression`, the expression is printed, but not evaluated.
 
-The first command assigns a value of Get-Process (a string) to the $Command variable.
+The first command assigns a value of `Get-Process` (a string) to the `$Command` variable.
 
 The second command shows the effect of typing the variable name at the command line.
-Windows PowerShell echoes the string.
+PowerShell echoes the string.
 
-The third command uses **Invoke-Expression** to evaluate the string.
+The third command uses `Invoke-Expression` to evaluate the string.
 
 ### Example 2: Run a script on the local computer
-```
-PS C:\> Invoke-Expression -Command "C:\ps-test\testscript.ps1"
-PS C:\> "C:\ps-test\testscript.ps1" | Invoke-Expression
+
+```powershell
+Invoke-Expression -Command "C:\ps-test\testscript.ps1"
+"C:\ps-test\testscript.ps1" | Invoke-Expression
 ```
 
-These commands use **Invoke-Expression** to run a script, TestScript.ps1, on the local computer.
+These commands use `Invoke-Expression` to run a script, TestScript.ps1, on the local computer.
 The two commands are equivalent.
-The first uses the *Command* parameter to specify the command to run.
-The second uses a pipeline operator (|) to send the command string to **Invoke-Expression**.
+The first uses the **Command** parameter to specify the command to run.
+The second uses a pipeline operator (|) to send the command string to `Invoke-Expression`.
```

Comment:
Should the pipeline operator be in a codefence?

Reply:
Fixed